webapp/components/sidebar_right_menu.jsx

```diff
@@ -173,6 +173,7 @@ export default class SidebarRightMenu extends React.Component {
         let joinAnotherTeamLink;
         let isAdmin = false;
         let isSystemAdmin = false;
+        const leaveTeamIcon = Constants.LEAVE_TEAM_SVG;
```

Comment:
Instead of assigning to a variable, it might be better to use the constant directly.